bedrock/firefox/templates/firefox/developer/includes/for-developers.html

```diff
@@ -13,7 +13,7 @@
 ) %}
   <h2 class="mzp-u-title-xs">{{ ftl('firefox-developer-firefox-developer-edition') }}</h2>
   <h3 class="mzp-u-title-sm">{{ ftl('firefox-developer-made-for-developers') }}</h3>
-  <p>{{ ftl('firefox-developer-all-the-latest-v2', fallback='firefox-developer-all-the-latest') }}</p>
+  <p>{{ ftl('firefox-developer-all-the-latest-v2') }}</p>
```

Comment:
In theory they still exist in a few locales: [@mozilla-l10n/www-l10n: `"firefox-developer-all-the-latest"`](https://github.com/search?q=repo%3Amozilla-l10n%2Fwww-l10n+firefox-developer-all-the-latest&type=code) but:
- `da` & `th` already have the new string anyways
- `be` is the only one not having the new string — so this one will default to _en_, but better than refer to (outdated) experimental features etc. — but if it's gone from _en_, it's gone…